docs/refs.bib

```diff
@@ -0,0 +1,7 @@
+@BOOK{fried:1961,
+       author = {{Fried}, Burton~D. and {Conte}, Samuel~D.},
+        title = "The Plasma Dispersion Function: The Hilbert Transformation of the Gaussian",
```

Comment:
This reminds me of my fanfic novel entitled: *Harry Potter and the Hilbert Transformation of the Gaussian*.